src/macaron/__main__.py

```diff
@@ -221,6 +217,26 @@ def main(argv: list[str] | None = None) -> None:
         If ``argv`` is ``None``, argparse automatically looks at ``sys.argv``.
         Hence, we set ``argv = None`` by default.
     """
+    # Handle presence of token file.
+    token_file = "./.macaron_env_file"  # nosec B105
```

Comment:
Instead of a hard-coded path, we can use `os.getcwd()`. Using `/` makes the implementation platform-specific. Instead we can let the `os` package to resolve the path.

Also, please add documentation on how this file is created and the difference in setting token when running Macaron as a container vs. Python package.

```suggestion
    token_file = os.path.join(os.getcwd(), ".macaron_env_file")
```

Looks like with this change Bandit doesn't report a FP hard-coded password error, so we can remove the [`# nosec B105`](https://bandit.readthedocs.io/en/latest/plugins/b105_hardcoded_password_string.html) comment.